backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/BpkDialog.kt

```diff
@@ -144,4 +146,52 @@ fun BpkDestructiveDialog(
   )
 }
 
+@Composable
+fun BpkFlareDialog(
+  onDismissRequest: () -> Unit,
+  title: String,
+  text: String,
+  confirmButton: DialogButton,
+  secondaryButton: DialogButton? = null,
+  properties: DialogProperties = DialogProperties(),
+  flareContent: @Composable BoxScope.() -> Unit,
```

Comment:
this follows the same API/properties as the other dialog functions, but with the flare content instead of icon